spec/forms/waste_exemptions_engine/location_form_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module WasteExemptionsEngine
+  RSpec.describe LocationForm, type: :model do
+    subject(:form) { build(:location_form) }
+
+    LOCATIONS = %w[
+      england
+      northern_ireland
+      scotland
+      wales
+    ].freeze
```

Comment:
Just occurred to me – if we have `frozen_string_literal: true` at the top, do we need to be explicitly freezing `LOCATIONS` and others like it? I don't think it hurts, but also my understanding is that the comment makes it unnecessary.

Reply:
I think Rubocop is doing this. I'll quickly check.